core/plugins/org.polarsys.capella.core.platform.sirius.ui.navigator/src/org/polarsys/capella/core/platform/sirius/ui/navigator/actions/providers/RepresentationActionProvider.java

```diff
@@ -26,11 +27,12 @@
 import org.eclipse.ui.navigator.ICommonMenuConstants;
 import org.eclipse.ui.navigator.ICommonViewerSite;
 import org.eclipse.ui.navigator.ICommonViewerWorkbenchSite;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
 import org.polarsys.capella.core.platform.sirius.ui.navigator.actions.CloneAction;
 import org.polarsys.capella.core.platform.sirius.ui.navigator.actions.SelectionHelper;
-import org.polarsys.capella.core.platform.sirius.ui.navigator.actions.move.representation.MoveRepresentationMenuManager;
```

Comment:
this class was also used in org.polarsys.capella.core.explorer.activity.ui\src\org\polarsys\capella\core\explorer\activity\ui\viewer\CapellaDiagramViewer
can you use your new class in it too.